### PR TITLE
chapter_8

### PR DESCRIPTION
**Pull request by ueihgnurt**
Screenshot from 2020-07-10 16-25-18

### CODE REVIEW

**Comment by ueihgnurt** [comment]
Ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -5,14 +5,13 @@ def new
 
   def show
     @user = User.find_by(id: params[:id])
-    if @user.blank?
-      redirect_to signup_path
-    end
+    redirect_to signup_path if @user.blank?
```
Comment (vanvtt-0952) [inline app/controllers/users_controller.rb:8]: chị chỉnh line code 8 một tí nhé
`return if @user`
`flash[:warning] = "User not found"`
`redirect_to signup_path`

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    User.find_by id: session[:user_id] if session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.nil?
```
Comment (vanvtt-0952) [inline app/helpers/sessions_helper.rb:11]: ` !current_user.nil?` thành `current_user.present?`

**Comment by vanvtt-0952** [comment]
merged
fix 2 comment rồi mợt nhé

**Pull request merged**